### PR TITLE
feat(api): use akash-api protobuf instead of akashjs (#184)

### PR DESCRIPTION
**Pull request by ygrishajev**
### Summary
This PR updates our project's approach to managing dependencies and configurations. We are transitioning to using `akash-api` protobuf-generated files.

### Changes
1. **Protobuf Integration**: Transitioned to using `akash-api` protobuf-generated files instead of `akashjs`. This shift ensures alignment with the latest specifications and streamlines our data models.
   
2. **TypeScript Configuration**: Updated `tsconfig.json` to include the `akash-api` package exports. This enhancement simplifies module resolution and optimizes our TypeScript environment.
   
3. **Local Testing Setup**: Temporarily included a zipped version of the `akash-api` package locally to facilitate passing CI tests. This measure will remain until the package is available on npm, allowing uninterrupted progress in development and testing.

### Notes
- This PR will be in draft status until `akash-api` is available on npm. This strategy prevents external dependencies from hindering our development and testing processes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -68,6 +68,7 @@ web_modules/
 
 # Output of 'npm pack'
 *.tgz
+!*/akashnetwork-akash-api-0.0.0.tgz
```
Comment (ygrishajev) [inline .gitignore]: To be removed when `akash-api` and next version of `akashjs` are published
```suggestion
```

File: api/package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "url": "git+https://github.com/akash-network/cloudmos"
   },
   "dependencies": {
+    "@akashnetwork/akash-api": "file:./akashnetwork-akash-api-0.0.0.tgz",
```
Comment (ygrishajev) [inline api/package.json]: To be updated when `akash-api` is published
```suggestion
    "@akashnetwork/akash-api": "@akashnetwork/akash-api@^1.0.0",
```

File: deploy-web/package.json [hunk 3]
```diff
@@ -15,8 +15,10 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
+    "@akashnetwork/akash-api": "file:./akashnetwork-akash-api-0.0.0.tgz",
```
Comment (ygrishajev) [inline deploy-web/package.json]: To be updated when `akash-api` is published
```suggestion
    "@akashnetwork/akash-api": "@akashnetwork/akash-api@^1.0.0",
```

File: indexer/package.json [hunk 4]
```diff
@@ -19,6 +19,7 @@
   },
   "homepage": "https://github.com/akash-network/cloudmos",
   "dependencies": {
+    "@akashnetwork/akash-api": "file:./akashnetwork-akash-api-0.0.0.tgz",
```
Comment (ygrishajev) [inline indexer/package.json]: To be updated when `akash-api` is published
```suggestion
    "@akashnetwork/akash-api": "@akashnetwork/akash-api@^1.0.0",
```

File: api/akashnetwork-akash-api-0.0.0.tgz
Comment (ygrishajev) [inline api/akashnetwork-akash-api-0.0.0.tgz:1]: To be removed when `akash-api` and next version of `akashjs` are published
```suggestion
```

File: deploy-web/akashnetwork-akash-api-0.0.0.tgz
Comment (ygrishajev) [inline deploy-web/akashnetwork-akash-api-0.0.0.tgz:1]: To be removed when `akash-api` and next version of `akashjs` are published
```suggestion
```

File: indexer/akashnetwork-akash-api-0.0.0.tgz
Comment (ygrishajev) [inline indexer/akashnetwork-akash-api-0.0.0.tgz:1]: To be removed when `akash-api` and next version of `akashjs` are published
```suggestion
```

File: deploy-web/Dockerfile [hunk 5]
```diff
@@ -8,6 +8,7 @@ FROM base AS deps
 WORKDIR /app
 
 # Install dependencies based on the preferred package manager
+COPY akashnetwork-akash-api-0.0.0.tgz .
```
Comment (ygrishajev) [inline deploy-web/Dockerfile]: To be removed when `akash-api` is published
```suggestion
```

**Review by Redm4x — APPROVED** [review]

File: deploy-web/tsconfig.json [hunk 6]
```diff
@@ -16,7 +16,7 @@
     "emitDecoratorMetadata": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```
Comment (baktun14) [inline deploy-web/tsconfig.json:19]: Interesting, just read about it quickly. I guess it gives better bundling performance?

**Review by baktun14 — APPROVED** [review]

File: deploy-web/tsconfig.json (hunk 6 above)
Comment (ygrishajev) [reply deploy-web/tsconfig.json:19]: I guess so. I primarily changed it to make sure packages' exports are supported and next requires this value

**Comment by baktun14** [comment]
No conventional commit? :)

**Comment by ygrishajev** [comment]
> No conventional commit? :)

hehe, it's an older PR) will update

**Pull request merged**